### ISSUE TITLE
gtk3 build error with 0.3.9

### ISSUE THREAD

**Issue by raveit65** [assen-totin/mate-applet-streamer#7]
With new release i get this compile issue with gtk3.

```
settings.c: In function 'menu_cb_settings':
settings.c:74:2: error: unknown type name 'GtkObject'
  GtkObject *adjustment = gtk_adjustment_new (initial_duration, 1, 60, 1, 10, 10);
  ^
settings.c:74:26: warning: initialization from incompatible pointer type [-Wincompatible-pointer-types]
  GtkObject *adjustment = gtk_adjustment_new (initial_duration, 1, 60, 1, 10, 10);
                          ^
settings.c:95:108: warning: not enough variable arguments to fit a sentinel [-Wformat=]
  applet->quitDialog = gtk_dialog_new_with_buttons (_("Settings"), GTK_WINDOW(applet), GTK_DIALOG_MODAL, NULL);
```

With gtk2 all is fine.
thx, for the new features.

**Comment by assen-totin** [comment]
Yes, my mistake. Fixed in 0.3.10 yesterday, but I did not know you already build the applet with GTK3 too. It's here: http://www.zavedil.com/wp-content/uploads/2016/01/mate-applet-streamer-0.3.10.tar.gz

**Comment by raveit65** [comment]
Thanks,
the gtk3 version you will find in the mate gtk3 repo since 0.3.2  ;)
https://copr.fedoraproject.org/coprs/raveit65/Mate-GTK3/
